docs/recording.md

```diff
@@ -37,3 +37,35 @@
         + `400 Bad Request` Message: `call`. Call is not activated.
         + `403 Forbidden` When the user is not a moderator/owner.
         + `412 Precondition Failed` When the lobby is active and the user is not a moderator.
+
+## Store call recording
+
+* Required capability: `recording-v1`
+* Method: `POPST`
+* Endpoint: `/recording/{token}/store`
+
+* Header:
+| field                     | type   | Description                                                                                                               |
+| ------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------- |
+| `TALK_SIPBRIDGE_RANDOM`   | string | Random string that need to be concatenated with room token to generate the checksum using the `sip_bridge_shared_secret`. |
+| `TALK_SIPBRIDGE_CHECKSUM` | string | The checksum generated with `TALK_SIPBRIDGE_RANDOM`.                                                                      |
```

Comment:
@nickvergessen I didn't found other place documenting these headers. Do we have other way or place to document this?